core/dictgen/src/rootcling_impl.cxx

```diff
@@ -3821,7 +3814,7 @@ int RootClingMain(int argc,
           && strcmp(argv[ic], "-f") != 0  && strcmp(argv[ic], "-r") != 0)  {
       if (!strcmp(argv[ic], "-l")) {
          ic++;
-      } else if (!strcmp(argv[ic], "-1")) {
```

Comment:
The new flag name is much better.  Do we know if any users were creating their own pch?

Reply:
I am not sure if we had any. Using an 'externally' created pch in ROOT is super tricky. If somebody is doing that it's quite likely they are using the same infrastructure as we do (the same script).

Reply:
fair enough. :)